Artifacts/linux-c++ethereum/installCPPEth.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# print commands and arguments as they are executed
+set -x
+
+echo "initializing (++)Ethereum installation"
+date
+ps axjf
+
+#############
+# Parameters
+#############
+
+AZUREUSER=$1
+HOMEDIR="/home/$AZUREUSER"
+VMNAME=`hostname`
+echo "User: $AZUREUSER"
+echo "User home dir: $HOMEDIR"
+echo "vmname: $VMNAME"
+
+# Fetch Genesis and scripts
+cd $HOMEDIR
+wget https://raw.githubusercontent.com/Azure/azure-devtestlab/master/Artifacts/linux-c++ethereum/lab-seed.json
+
+#####################
+# setup the Azure CLI
+#####################
+time sudo npm install azure-cli -g
+time sudo update-alternatives --install /usr/bin/node nodejs /usr/bin/nodejs 100
+
+####################
+# Setup (++)Ethereum
+####################
+sudo add-apt-repository ppa:ethereum/ethereum-qt
+sudo add-apt-repository ppa:ethereum/ethereum
+sudo apt-get update
+sudo apt-get install cpp-ethereum
+
+# Fetch Genesis and Private Key
+cd $HOMEDIR
+
+date
+echo "completed (++)Ethereum install $$"
```

Comment:
Disable trace printing at the conclusion of the script to avoid any downstream issues with other scripts. (i.e. set +x)